Source/MASFlightComputerProxy2.cs

```diff
@@ -1548,7 +1548,8 @@ public double RecoverVessel()
         {
             if (vessel.IsRecoverable)
             {
-                GameEvents.OnVesselRecoveryRequested.Fire(vessel);
+                AltimeterSliderButtons sliderButtons = (AltimeterSliderButtons)GameObject.FindObjectOfType(typeof(AltimeterSliderButtons));
+                sliderButtons.vesselRecoveryButton.onClick.Invoke();
```

Comment:
Can this trigger a null-reference exception (are there situations where it returns null)? In issue #209 you mentioned `ShipConstruction.RecoverVesselFromFlight()` - have you looked at that method as a recovery option?

Reply:
I tried RecoverVesselFromFlight, and it doesn't do the right thing - I think it's a sub-step in the recovery process, but doesn't start the process.  Based on the deeper investigation I've been doing I think this might entirely be an issue within ProbeControlRoom.